packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -121,6 +121,7 @@ class ProcessSignal implements io.ProcessSignal {
   static const ProcessSignal SIGUSR1 = const _PosixProcessSignal._(io.ProcessSignal.SIGUSR1); // ignore: deprecated_member_use
   static const ProcessSignal SIGUSR2 = const _PosixProcessSignal._(io.ProcessSignal.SIGUSR2); // ignore: deprecated_member_use
   static const ProcessSignal SIGINT =  const ProcessSignal(io.ProcessSignal.SIGINT); // ignore: deprecated_member_use
```

Comment:
Why not use the non-deprecated versions (sigint and sigkill, etc.)?

Reply:
The others were already there with the comments and I wasn't entirely sure if there was a reason (like if this code needed to be backward-compatible with something). If not, I'm happy to fix; it did bug me adding that comment :-)

Reply:
Well, it's probably a bigger change to convert the declarations of the uppercase constants we have in `io.dart` to lowercase (since things might be using them), but it's pretty trivial to change their definitions to use the lowercase constants (and remove the exception comments).

This is just a result of Dart deprecating uppercase constants in favor of their lowercase equivalents: the values of the constants remain the same, so the definitions won't change.

Reply:
Yeah, makes sense - I pushed that change :)